cmd/limactl/main.go

```diff
@@ -31,7 +32,34 @@ const (
 	advancedCommand     = "advanced"
 )
 
+// yqmain is a copy of main() in https://github.com/mikefarah/yq/blob/master/yq.go
```

Comment:
Better to split a package, with an appropriate SPDX headers

Reply:
What is the appropriate license header? I think it needs to maintain the MIT license and include the original copyright in addition to ours.